### PR TITLE
LAMP Scripts

### PR DESCRIPTION
**Pull request by rennergade**
I reorganized the whole scripts folder to easily install all the lamp components as well as updated a few things (added some python site-packages, fixed the conf file, added the grp module to python. This should make testing the LAMP much easier.

### CODE REVIEW

File: src/scripts/lamp/load_lamp.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+/home/lind//lind_project/src/scripts/base/load_bash.sh
+/home/lind//lind_project/src/scripts/base/load_coreutils.sh
+/home/lind//lind_project/src/scripts/base/load_confs.sh
+
+/home/lind//lind_project/src/scripts/python/load_python.sh
+/home/lind//lind_project/src/scripts/nginx/load_nginx.sh
+/home/lind//lind_project/src/scripts/postgres/load_postgres.sh
+/home/lind//lind_project/src/scripts/psycopg2/load_psycopg2.sh
```
Comment (jesings) [inline src/scripts/lamp/load_lamp.sh:10]: Get rid of the double slashes

File: src/scripts/lamp/compile_lamp.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+/home/lind//lind_project/src/scripts/base/compile_bash.sh
+/home/lind//lind_project/src/scripts/base/compile_coreutils.sh
+/home/lind//lind_project/src/scripts/base/compile_confs.sh
+
+/home/lind//lind_project/src/scripts/python/compile_python.sh
+/home/lind//lind_project/src/scripts/nginx/compile_nginx.sh
+/home/lind//lind_project/src/scripts/postgres/compile_postgres.sh
+/home/lind//lind_project/src/scripts/psycopg2/compile_psycopg2.sh
+
```
Comment (jesings) [inline src/scripts/lamp/compile_lamp.sh:11]: Get rid of the double slashes

**Review by jesings — APPROVED** [review]
Overall looks good, one small nitpick

**Review by Devansh-Patel — APPROVED** [review]
Looks good to me.

**Pull request merged**